src/status.js

```diff
@@ -0,0 +1,47 @@
+function setStyle(json) {
+  const inputs = document.getElementsByClassName('text');
+  const chks = document.getElementsByClassName('check');
+  const obj = JSON.parse(json);
+
+  obj.forEach((element) => {
+    if (element.checked === true) {
+      inputs[element.index].classList.add('done');
+      chks[element.index].checked = true;
+    } else {
+      inputs[element.index].classList.remove('done');
+    }
+  });
+}
+function setLocalStorage() {
+  const arr = [];
```

Comment:
- Kindly fix this error in chrome console: `` Uncaught TypeError: Cannot read property 'classList' of undefined ``.